src/components/FilterListItem/FilterListItem.jsx

```diff
@@ -21,15 +21,15 @@ class FilterListItem extends React.Component {
               className="button is-link"
               title="Edit filter"
             >
-              <span className="octicon octicon-pencil"></span>
+              ✏️
```

Comment:
Since this is GitHub specific app, it might be best to use Octicons for everything